devolving_music/views/current_playlist.py

```diff
@@ -0,0 +1,14 @@
+from django.views import View
+
+from devolving_music.models.event import Event
+from devolving_music.lib.song_scores import SongScores
+from .param_utils import success, safe_url_params
+
+
+class CurrentPlaylist(View):
+    @safe_url_params
+    def get(self, _request, event: Event):
+        return success([
+            score.to_json()
+            for score in SongScores(event).get_final_list()
```

Comment:
It's a bit funny that this method is still called `get_final_list`, but I didn't want to change code you wrote without good reason, and anyway this list is still "final" in the sense of being the final outcome of an algorithmic process, which involved multiple other song lists.